src/resty/ljsonschema/init.lua

```diff
@@ -946,6 +949,8 @@ generate_validator = function(ctx, schema)
         ctx:stmt(sformat('  %s == %q', ctx:param(1), val), op)
       elseif tval == 'table' then
         ctx:stmt(sformat('  %s(%s, %s)', ctx:libfunc('lib.deepeq'), ctx:param(1), ctx:uservalue(val)), op)
+      elseif tval == 'userdata' and val == default_null then
+        ctx:stmt(sformat(' %s == default_null', ctx:param(1), val), op)
       else
         error('unsupported enum type: ' .. tval) -- TODO: null
```

Comment:
The `TODO: null` comment should be removed?

Reply:
done